### DISCUSSION TITLE
serialized size compared with protobuf?

### DISCUSSION THREAD

**Discussion by sprappcom** [capnproto/capnproto#2096] [Q&A]
any size comparison? is protobuf smaller in serialized size generated?

**Comment by jclee** [comment]
Looks like there's some size comparison info in this discussion thread?:

https://groups.google.com/g/capnproto/c/QsMRreEdPpU

(from 2014, but I think still applies to current implementations.)

**Reply by kentonv** [reply]
Yep, that's still accurate.
